_posts/2019-03-04-jumping-on-the-dash-less-bandwagon.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title:  "Jumping on the '-less' Bandwagon"
+date:   2019-03-04
+tags: serverless
+author: Pete Yandell
+image: img/bandwagonless/bandwagonless.png
+---
+
+<img style="display: block; margin-left: auto; margin-right: auto; width: 40%;" alt="wagon wheel" src="{{ site.url }}/img/bandwagonless/bandwagonless.png"/>
+
+## Introduction
+
+It has been awhile since I made my [last post](https://mechanicalrock.github.io/2018/08/27/inception-pipelines-pt6), the one where at the end I promised something about the [AWS CDK](https://awslabs.github.io/aws-cdk/) and how awesome it was going to be. Since then a few of my colleagues have mastered it, and I got super excited by something else.
+
+That something else is SER-VER-LESS!! And so now, I to, am _Jumping on the '-less' Bandwagon!_
+
+## What is serverless and why all the hype?
+
+If you scour the Internet, you will find as many definitions as there are search results. Most of them dive into esoteric technical definitions of what is and isn’t serverless.
+
+My technical perspective is that I don’t consider building, running and managing containers as serverless. I also don't count no-infrastructure pieces of your architecture like DynamoDB, Identity management, payment gateways, etc as Serverless either. They are however critical ‘no-ops’ pieces to operating Serverlessly. What’s left is really the functions-as-a-service services like AWS Lambda, Google Functions and Azure Functions. These functions strip away everything that are not critical core features that are unique to your business (versus structural piece like databases)
```

Comment:
I know this is _your_ perspective, but I disagree with this statement:
>I also don't count no-infrastructure pieces of your architecture like DynamoDB, Identity management, payment gateways, etc as Serverless either.

To me, serverless is more than just anonymous functions that do stuff. It's a service that's provided without the need to manage servers (or in AWS's case at least, you flat out don't have the ability to).

Reply:
lol  "Hamishs' LI/twitter ramblings etc"